static/app/components/discover/transactionsList.tsx

```diff
@@ -176,7 +175,13 @@ class _TransactionsList extends Component<Props> {
     return (
       <Fragment>
         <div>
-          <DropdownControl
+          <CompactSelect
+            triggerProps={{prefix: t('Filter'), size: 'xsmall'}}
+            value={selected.value}
+            options={options}
+            onChange={opt => handleDropdownChange(opt.value)}
+          />
+          {/* <DropdownControl
```

Comment:
Is there a reason for leaving the old component commented out?

Reply:
Oops I forgot to delete this – had not had enough coffee lol 😅